change/@uifabric-foundation-2019-11-06-08-57-35-jg-deprecate-foundation-type.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Deprecate IPropsWithChildren type that is redundant with React.PropsWithChildren.",
```

Comment:
```suggestion
  "comment": "Foundation: Deprecate IPropsWithChildren type that is redundant with React.PropsWithChildren.",
```

Reply:
The change file is already scoped to the foundation package. I think this is more useful for intra-package designation.